src/site/apt/user_indexing.apt

```diff
@@ -44,14 +44,14 @@ If I need to create index for xml collection stored <path_to_collection>/collect
 
 Query structure:
 --------
-build-index-on-collection("<path_to_collection>")
+build-index-on-collection("collection")
```

Comment:
How about using "<path_to_collection>" or "<path_to_collection_1>" and "<path_to_collection_2>"?

The example paths in this document are not consistent.